css/ai.css

```
@@ -228,4 +204,61 @@
     }
 }
 
-@tailwind utilities;
\ No newline at end of file
+#content a.button-next-page {
+    @apply btn btn-primary text-primary-content font-medium shadow-md shadow-accent pt-0 pb-0 pr-8 pl-8 no-underline 
+}
+
+#content h4 {
+    @apply text-xl font-medium mt-6 mb-3 text-primary dark:text-secondary
+}
+
+#content p {
+    @apply leading-6 font-normal text-inherit
+}
+
+.inner {
+    @apply prose pl-5 pr-5 md:pl-0 md:pr-0 sm:max-w-3xl text-black dark:text-white bg-base-100 my-auto mx-auto dark:bg-base-100
+}
+
+#content h2 {
+    @apply text-3xl font-medium mt-9 mb-5 text-inherit
+}
+
+/* blue bar above the page headers*/
+#content h2::before {
+    content: '';
+    display: block;
+    height: 0.3125rem;
+    @apply border-t-4 border-primary my-auto w-16 mb-2;
+    border-color: #4579FF; /* MH: not sure why tailwind adds some weird HSL value that overrides our primary color */
+}
+
+#newsletter-section {
+    @apply py-6;
+}
+
+
+.email-checkbox {
+    width: 24px; height: 24px;
+}
+
+.email-submit {
+    /* MH: not sure why tailwind adds some weird HSL value that overrides our primary color */
+    @apply text-neutral;
+}
+
+#newsletter-thanks {
+    display: none;
```

Not sure if we can use Tailwind utility classes here:
```suggestion
    @apply hidden;
```

https://tailwindcss.com/docs/display

Class | Properties
:----|:----
`hidden` | `display: none;`